<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width> <height>850</height> </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>POWERGYM - Registro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../recursos.qrc">
    <normaloff>:/logo/514ba084-0d56-41a1-9273-abc5a52fc600.png</normaloff>:/logo/514ba084-0d56-41a1-9273-abc5a52fc600.png</iconset>
  </property>
  <property name="styleSheet"> <string notr="true">
    QMainWindow#MainWindow { /* Usamos el objectName de la ventana principal */
     background-image: url(:/Imagenes/descarga.jpg); /* Asegúrate de que esta ruta es correcta en recursos.qrc */
     background-repeat: no-repeat;
     background-position: center;
     background-size: cover;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_left">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame_registro_principal">
      <property name="minimumSize">
       <size>
        <width>450</width> <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        QFrame#frame_registro_principal {
         background-color: rgba(255, 255, 255, 180); /* Blanco con 70% de opacidad */
         border-radius: 15px;
         padding: 10px; /* Padding para el frame */
        }
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_registro">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_titulo_registro">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(55,5,255);</string>
         </property>
         <property name="text">
          <string>REGISTRO DE USUARIO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_nombre">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nombre">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="placeholderText">
          <string>Ingrese su nombre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_apellidos">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Apellidos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_apellidos">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="placeholderText">
          <string>Ingrese sus apellidos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_email">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Correo Electrónico:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_email">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="placeholderText">
          <string>ejemplo@dominio.com</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_password">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Contraseña:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_password">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Ingrese su contraseña</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_confirm_password">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Confirmar Contraseña:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_confirm_password">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Confirme su contraseña</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_fecha_nacimiento">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fecha de Nacimiento:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateedit_fecha_nacimiento">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QDateEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QDateEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_telefono">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Teléfono:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_telefono">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLineEdit {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QLineEdit:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="placeholderText">
          <string>Ej: 600123456</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_peso_corporal">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Peso Corporal (kg):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinbox_peso_corporal">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QDoubleSpinBox {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
           }
           QDoubleSpinBox:focus {
            border: 2px solid rgb(0, 0, 200);
           }
          </string>
         </property>
         <property name="suffix">
          <string> kg</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.01</double>
         </property>
         <property name="maximum">
          <double>300.00</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_middle_bottom_form">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_rol">
          <property name="font">
          <font>
            <pointsize>10</pointsize>
          </font>
          </property>
          <property name="text">
          <string>Tipo de usuario:</string>
          </property>
        </widget>
        </item>
        <item>
        <widget class="QComboBox" name="combo_rol">
          <property name="minimumSize">
          <size>
            <width>0</width>
            <height>40</height>
          </size>
          </property>
          <property name="font">
          <font>
            <pointsize>11</pointsize>
          </font>
          </property>
          <property name="styleSheet">
          <string notr="true">
            QComboBox {
            background-color: rgba(255, 255, 255, 200);
            border: 2px solid rgb(55, 5, 255);
            border-radius: 8px;
            padding: 8px;
            color: black;
            }
            QComboBox:focus {
            border: 2px solid rgb(0, 0, 200);
            }
          </string>
          </property>
          <property name="currentIndex">
          <number>-1</number>
          </property>
          <item>
          <property name="text">
            <string>Administrador</string>
          </property>
          </item>
          <item>
          <property name="text">
            <string>Entrenador</string>
          </property>
          </item>
          <item>
          <property name="text">
            <string>Atleta</string>
          </property>
          </item>
        </widget>
        </item>

       <item>
        <widget class="QPushButton" name="btn_registrar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QPushButton {
            background-color: #4CAF50; /* Verde, coherente con VistaInicial */
            color: white;
            border-radius: 15px;
            padding: 10px;
            border: none;
           }
           QPushButton:hover {
            background-color: #45a049;
           }
          </string>
         </property>
         <property name="text">
          <string>Registrar Cuenta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_volver_login">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize> </font>
         </property>
         <property name="text">
          <string>¿Ya tienes una cuenta? Inicia Sesión</string> </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QPushButton {
            color: rgb(55, 5, 255); /* Azul/Morado, coherente con las demás */
           }
           QPushButton:hover {
            text-decoration: underline;
           }
          </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_bottom_form">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_right">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>